com.microsoft.mrtk.input/Interactors/GazePinch/GazePinchInteractor.cs

```diff
@@ -38,7 +38,22 @@ public class GazePinchInteractor :
         /// <summary>
         /// The worldspace pose of the hand pinching point.
         /// </summary>
-        public Pose PinchPose => handController.PinchSelectPose;
+        public Pose PinchPose => pinchPoseSource.TryGetPose(out Pose pinchPose) ? pinchPose : Pose.identity;
+
+        [SerializeReference]
+        [InterfaceSelector]
+        [Tooltip("The pose source representing the ray this interactor uses for rotation")]
+        private IPoseSource rayPoseSource;
+
+        [SerializeReference]
+        [InterfaceSelector]
+        [Tooltip("A fallback pose source representing a palm this interactor uses for rotation")]
+        private IPoseSource fallbackPalmPoseSource;
```

Comment:
This is a pattern that I've been looking at as well, where it tries to query an input action and falls back to a specific hand joint if the input action isn't available. I wonder if we could rework this slightly to make that a common pattern...

I can also take a look at this in my follow-up PR

Reply:
I think we could make a "compound" pose source class. I'll draft one up for this PR.

Reply:
Made a compound class called `FallbackPoseInteractor` I couldn't really find the right word to describe a system which has a lot of redundancy built into it in the form of backups and fallbacks